lib/datasource/gitlab-packages/index.ts

```diff
@@ -0,0 +1,72 @@
+import URL from 'url';
```

Comment:
Don't use deprecated url. Checkout our util/url helpers.

Reply:
I'm ok to keep this *if* it's just a copy/paste from an existing datasource. We'll need to refactor those other ones anyway, and can do all the similar ones at the same time.

Reply:
@rarkins You told me to not make all those changes in one pr 😉 

So i would prefer to now use the right function from our utils. Should be a very small change.

Reply:
He already did the required change 🙃